README.md

```diff
@@ -1,42 +1,79 @@
-# Sensible logging
+# Sensible Logging
 
 [![CircleCI](https://circleci.com/gh/madetech/sensible_logging.svg?style=svg)](https://circleci.com/gh/madetech/sensible_logging) [![Gem Version](https://badge.fury.io/rb/sensible_logging.png)](http://badge.fury.io/rb/sensible_logging)
 
-
-A logging library with sensible defaults for Sinatra apps.
+A logging extension with sensible defaults for Sinatra apps.
 
 ## Features
 
-* Add (or use an existing if present `X-Request-Id`) request ID for use in logs
+* Add a request UUID (or use an existing one if present in the `X-Request-Id` HTTP header) for use in logs, your app or other middlewares
 * Trim the request logs to the bare minimal (inspired by lograge):
   * method
   * path
   * requesting IP address
   * status
   * duration
   * params if a `GET` request
+
+  Example log line:
+  ```
+  method=GET path=/contact client=192.168.1.254 status=200 duration=0.124 params={"category"=>"question"}
+  ```
 * Tagged logging, with some sensible defaults:
   * subdomain
   * environment
-  * unique request ID
+  * request UUID
+
+  Example log line:
+  ```
+  [www.gb] [staging] [6004bb70-7b6d-43b6-a2cf-72d0336663ba] @todo tidy sql query
+  ```
 
 ## Usage
 
-1. Add `sensible_logging` to your `Gemfile` and use `bundle install`.
-2. In `app.rb` register the module and then define your logging defaults.
+1. Add `sensible_logging` to your `Gemfile` and install with `bundle install`:
 
-```ruby
-require 'sensible_logging'
+  ```ruby
+  gem 'sensible_logging'
+  ```
+2. In your `app.rb` register the module and then define your logging configuration:
 
-class App < Sinatra::Base
-  register Sinatra::SensibleLogging
+  ```ruby
+  require 'sensible_logging'
 
-  sensible_logging(
-    logger: Logger.new(STDOUT)
-    )
+  class App < Sinatra::Base
+    # Register the extension
+    register Sinatra::SensibleLogging
 
-# rest of code omitted
-```
+    # Configure Sensible Logging
```

Comment:
this isn't just configuring. I think we might want to signify this is also doing final initialisation of the extension.

```suggestion
# Initialise SensibleLogging to add our middlewares.
```